cylc/flow/network/client.py

```diff
@@ -147,7 +147,7 @@ def timeout_handler(self) -> None:
         # behind a contact file?
         try:
             detect_old_contact_file(self.workflow)
-        except (AssertionError, ServiceFileError):
+        except ContactFileExists:
```

Comment:
Any idea why `AssertionError` was there?

Reply:
Did a quick git blame time travel, there used to be an `AssertionError` raised in `detect_old_contact_file()` but no more